.travis.yml

```
@@ -116,7 +116,9 @@ jobs:
     if: repo == "ansible/ansible-lint" AND tag IS present
     python: "3.7"
     env: &deploy-env
-      TOXENV: metadata-validation
+      TOXENV: metadata-validation,build-dists
```

The order of execution matters. First, build, then validate.
```suggestion
      TOXENV: build-dists,metadata-validation
```
Alternatively, we could move `tox -e metadata-validation` to `before_deploy` step.

---

Pro tip: it's important to accept suggested changes via GitHub UI since it adds a proper co-author commit trailers. You can then fetch such commit and do squashing once you know that metadata (it's not always public/easily accessible).